### DISCUSSION TITLE
Unable to load extension in edge browser

### DISCUSSION THREAD

**Discussion by bibinmohana** [webdriverio/webdriverio#8635] [Community]
Dear All , 
I am unable to load extensions in edge browser . I have tried adding user data path in to 'args:' , browser is launching but extension is not getting loaded . I am able to launch extension with the same approach in chrome. Can someone please help here.

Sample code tried :
'ms:edgeOptions':{
args:[user-data-dir=\path to userdata']
}

**Comment by VishnuSomisetty** [comment]
I am also facing same issue it is not working in edge browser. kindly requesting can someone please look into the issue

**Comment by erwinheitzman** [comment]
Have you tried forward slashes in the path?

Reference: https://forum.robotframework.org/t/chromium-edge-browser-arguments-user-data-dir/1723/6

**Reply by bibinmohana** [reply]
Thanks for the response .I have used backward slash only. I tried replicating the same scenario in my personal machine (sample url) and its working fine, Able to launch edge , firefox and chrome with modheader. The difference is corporate proxy and node js version .In my official laptop selenium-standalone-service is not getting installed properly , getting  warning unsupported engine with node version . Also while trying to execute , am getting 'Error in "getDownloadStream"

**Reply by bibinmohana** [reply]
Any idea why we are getting " Error in gerDownloadStream"  while downloading selenium server ? Is it due to corporate proxy ? or Node js version ? these are the only two difference that I could see .. Can you please help here. Thanks in advance.

**Reply by bibinmohana** [reply]
Hi All , when i updated the proxy using global agent , its downloading the selenium server and other drivers. But now am getting an error " Failed to connect to Selenium " Error @wdio/selenium-service : Error : unable to connect to seleniumlib\check-started.js

**Reply by erwinheitzman** [reply]
This sounds unrelated to the original question?

**Reply by erwinheitzman** [reply]
I do not know what Node version you are running but that might be the problem

**Reply by bibinmohana** [reply]
Node version is V16.9.0
